F-5

AniraUoren opened the pull request:
Сделан блок "Помогли клиентам..."

AniraUoren committed:
feat: Верстка блока "Что входит в программу"

1. Добавлены изображения блока.
2. Добавлены все необходимые
переменные.
3. Сделан миксин для удобного указания шрифта.
4. Добавлена
верстка в index.html
5. Написаны стили под десктоп.

Closes F-4

AniraUoren committed:
fix: Поправлены стили

Closes F-4

AniraUoren committed:
feat: Добавлены стили для планшета и мобилки

Closes F-4

AniraUoren committed:
feat: Сверстан блок партнеров

Сделаны все разрешения.

Closes F-5

AniraUoren committed:
feat: Сверстан блок партнеров

Сделаны все разрешения.

Closes F-5

777toha commented:
![image](https://github.com/EasyDo-project/easydo-webpack/assets/110735886/da1597f4-2308-4346-a148-dadffa6a4998)

отсутствуют файлы

777toha commented on src/pages/index.html:
```diff
@@ -16,6 +16,120 @@
 	<body class="page">
-		<h1>Hello world!</h1>
+		<section class="assist">
+			<h2 class="assist__header">Помогли клиентам более 30 клиник</h2>
```
Весь текст вынести в константы и импортировать сюда

777toha commented on src/pages/index.html:
```diff
@@ -16,6 +16,120 @@
+			<div class="assist__container">
+				<img
+					src="<%=require('../images/assist_1.png')%>"
+					alt=""
```
alt обязательны

AniraUoren committed:
fix: Правки в блок помогли.

1. Добавлены шрифты.
2. Заголовок блока тянется из констант.
3. Добавлены рыбные alt для логотипов.

Closes F-5

AniraUoren commented:
Внесла правки

777toha commented on src/pages/index.html:
```diff
@@ -13,9 +13,123 @@
+				<img
+					src="<%=require('../images/assist_15.png')%>"
+					alt="Clinic"
+					class="assist__image"
+				/>
+				<img
+					src="<%=require('../images/assist_16.png')%>"
+					alt="Clinic"
+					class="assist__image"
+				/>
+				<img
+					src="<%=require('../images/assist_17.png')%>"
+					alt="Clinic"
+					class="assist__image"
+				/>
+				<img
+					src="<%=require('../images/assist_18.png')%>"
+					alt="Clinic"
+					class="assist__image"
+				/>
+				<img
+					src="<%=require('../images/assist_19.png')%>"
+					alt="Clinic"
+					class="assist__image"
+				/>
+				<img
+					src="<%=require('../images/assist_20.png')%>"
+					alt="Clinic"
+					class="assist__image"
+				/>
+				<img
+					src="<%=require('../images/assist_21.png')%>"
+					alt="Clinic"
+					class="assist__image"
+				/>
+				<img
+					src="<%=require('../images/assist_22.png')%>"
+					alt="Clinic"
+					class="assist__image"
+				/>
```
В целом к этой ветке вопросов нету, только один вопрос, повторяющиеся блоки не круто, сможешь так же пройтись foreach что бы картинки сами подставлялись и вынести это в js?

777toha commented:
если не получиться, я приму и так

777toha committed:
Merge branch 'dev' into feature/F-5
